apps/src/code-studio/components/ShareAllowedDialog.jsx

```diff
@@ -25,11 +25,28 @@ import Button from '../../templates/Button';
 import Spinner from '@cdo/apps/code-studio/pd/components/spinner';
 import defaultThumbnail from '@cdo/static/projects/project_default.png';
 import fontConstants from '@cdo/apps/fontConstants';
+import analyticsReporter from '@cdo/apps/lib/util/AnalyticsReporter';
+import {EVENTS} from '@cdo/apps/lib/util/AnalyticsConstants';
+
+const SHARE_TYPE_EVENT = {
+  'open': EVENTS.SHARING_DIALOG_OPEN,
+  'copy': EVENTS.SHARING_LINK_COPY,
+  'publish': EVENTS.SHARING_PUBLISH,
+  'facebook': EVENTS.SHARING_FB,
+  'twitter': EVENTS.SHARING_TWITTER,
+  'send-to-phone': EVENTS.SHARING_LINK_SEND_TO_PHONE,
```

Comment:
Are these keys (eg, `send-to-phone`) used anywhere else? Could we just use the event string (eg, `EVENTS.SHARING_LINK_SEND_TO_PHONE`) as the `type` arg in the calls to `wrapShareClick`?

Reply:
Good call! Using the `EVENTS` key names.